trace-forward/src/Trace/Forward/Forwarder.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE LambdaCase #-}
+
+-- This top-level module will be used by the forwarder application.
+-- Forwarder application collects 'TraceObject's and sends them to
+-- the acceptor application.
+module Trace.Forward.Forwarder
+  ( runTraceForwarder
+  ) where
+
+import qualified Codec.Serialise as CBOR
+import           Control.Concurrent.STM.TBQueue (TBQueue)
+import           Control.Exception.Extra (try_)
+import           Data.Typeable (Typeable)
+import           System.Time.Extra (sleep)
+
+import           Ouroboros.Network.Util.ShowProxy (ShowProxy(..))
+
+import           Trace.Forward.Configuration (ForwarderConfiguration (..))
+import           Trace.Forward.Network.Forwarder (connectToAcceptor)
+
+runTraceForwarder
+  :: (CBOR.Serialise lo,
+      ShowProxy lo,
+      Typeable lo)
+  => ForwarderConfiguration lo -- ^ Forwarder configuration.
+  -> TBQueue lo                -- ^ The queue the forwarder will take 'TraceObject's from.
```

Comment:
This is okay, but I think "The queue from which the forwarder will take 'TraceObject's" sounds a bit better.